src/scenes/game.js

```diff
@@ -19,7 +19,8 @@ export default class GameScene extends Phaser.Scene {
         });
     }
 
-    create() {
+    create(data) {
+        score = data.score ? data.score : 0;
```

Comment:
Also, now that you're always setting `score` to default to `0` here, you can remove the `= 0;` from above. It's unnecessary now _(I think? right?)_

Reply:
Agree. I'll remove

Reply:
I'm wondering when `score` is not 0 at the start of a `GameScene`